test/support/builders/metadata/base.js

```diff
@@ -69,7 +69,7 @@ module.exports = class BaseMetadataBuilder {
   }
 
   unmerged (sideName /*: SideName */) /*: this */ {
-    delete this.sides
+    if (this.doc.docType === 'file') delete this.doc.sides
```

Comment:
Why not for a `folder`?

Reply:
To match `buildDir`'s implementation:
> Also, when building a directory, we set the sides attribute to an
empty object instead of not having an attribute at all like we do when
building a file.

If we remove the `sides` attribute entirely, we won't have something equivalent to a directory Metadata object that has been built but not merged yet.